REVIEWING.md

```diff
@@ -0,0 +1,125 @@
+# Reviewing PRs at TensorZero
+
+We require a code review of every PR to the repository.
```

Comment:
Like we discussed, let's also try to establish an SLO for this, aiming for max ~1 business day turnaround?